app/middleware/error_handler.py

```
@@ -0,0 +1,512 @@
+            # Used as part of FastAPI middleware stack
+            app.add_middleware(ErrorHandlerMiddleware)
+        
+        Generated by CodeRabbit
+        """
+        try:
+            response = await call_next(request)
+            return response
+            
+        except Exception as exc:
+            # This should rarely be hit as exception handlers should catch most errors
+            logger.error(
```

Ruff (0.11.9)

Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)